content/pt/docs/referencia-da-api/pix/pagamento/listar-pagamentos/_index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Listar Pix"
+linkTitle: "Listar Pix"
+date: 2021-06-25T14:28:33-03:00
+lastmod: 2021-06-25T14:28:33-03:00
+weight: 1
+draft: false
+description: >
+  Como consultar pagamentos com Pix enviados
```

Comment:
Aqui serve tanto para pagamentos/devoluções enviadas e recebidas né? Se for, talvez seja melhor deixar mais explícito aqui, quando li eu achei que eram somente para pagamentos enviados.